ss/pebbledb/db.go

```diff
@@ -482,6 +516,25 @@ func prependStoreKey(storeKey string, key []byte) []byte {
 	return append(storePrefix(storeKey), key...)
 }
 
+// Parses store from key with format "s/k:{store}/..."
+func parseStoreKey(key []byte) (string, error) {
```

Comment:
nit: Alternatively, you can have this method use a regex. it'll be simpler and fast. Match on the first word after prefix and before `/`

Reply:
Will leave with string split for now